lib/streamlit/config.py

```diff
@@ -1211,8 +1220,45 @@ def func_with_lock():
     return disconnect
 
 
+def _check_static_files_config():
+    # Import logger locally to prevent circular references
+    from streamlit.logger import get_logger
+
+    LOGGER = get_logger(__name__)
+
+    # TODO [KAREN] Rephrase warning messages
+    if get_option("server.enableStaticServing"):
+
+        static_folder_path = os.path.abspath("static")
+
+        if not os.path.isdir(static_folder_path):
+            LOGGER.warning(
+                "STATIC FILE SERVING ENABLED, but folder static/ "
+                "doesn't exist :( :( :("
+            )
+        else:
+            try:
+                real_static_folder_path = file_util.sanitize_user_static_path(
+                    static_folder_path
+                )
+            except RuntimeError as e:
+                LOGGER.error(str(e))
+                set_option(
+                    "server.enableStaticServing",
+                    False,
+                    ConfigOption.STREAMLIT_DEFINITION,
+                )
+
+            else:
+                LOGGER.info(
+                    "STATIC FILE SERVING ENABLED FROM"
+                    " FOLDER %s" % real_static_folder_path
+                )
```

Comment:
We don't report any other positive messages for config settings at `info` or `debug` level from what I can tell. Given that, I would just remove this IMO.

If you want to keep it, maybe use `Static file serving enabled with 'server.enableStaticServing=true'. Files in folder './static' will be served on '/app/static'`.

Also note, I would not include the full path from root in the log since it could be a security/privacy issue depending where the app is hosted and how logs are being stored.